extensions/positron-assistant/package.json

```diff
@@ -590,5 +590,8 @@
     "@anthropic-ai/sdk": "^0.57.0",
     "@github/copilot-language-server": "^1.335.0",
     "vscode-languageclient": "^9.0.1"
-  }
+  },
+  "extensionDependencies": [
```

Comment:
So Assistant really does have to run in the same process, but the Jupyter extension doesn't? Do you know why? (Just for my own learning)